dm/unit/unit.go

```diff
@@ -65,7 +63,7 @@ type Unit interface {
 // we can refine to add error scope field if needed
 func NewProcessError(err error) *pb.ProcessError {
 	result := &pb.ProcessError{
-		Msg: errors.ErrorStack(err),
+		Msg: terror.Message(err),
```

Comment:
Can we set `Msg` to "" ? It's weired that users can get the similar message from `Msg` and `Error`.

Reply:
wait:joy:, In fact, I just changed it, but I still can't pass the test, and then rolled back some to verify which error was caused. Finally, it seems that it is necessary to modify the test case. :joy:

Reply:
And i was just deploying a test environment on my PC :joy: